Calibration/HcalCalibAlgos/src/RecAnalyzerMinbias.cc

```diff
@@ -35,58 +36,64 @@ class RecAnalyzerMinbias : public edm::EDAnalyzer {
 public:
   explicit RecAnalyzerMinbias(const edm::ParameterSet&);
   ~RecAnalyzerMinbias();
-  virtual void analyze(const edm::Event&, const edm::EventSetup&);
-  virtual void beginJob() ;
-  virtual void endJob() ;
+
+  virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
```

Comment:
This is still a "legacy" analyser. I think it is safe to move to "global" or "one".
To be revisited.